.github/workflows/post-release.yml

```diff
@@ -68,47 +32,41 @@ jobs:
 
   docker:
     name: Publish to GHCR
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
 
     permissions:
       contents: read
       packages: write
 
     steps:
-      - name: Checkout ${{ github.event.release.tag_name || github.event.inputs.tag_name }}
+      - name: Checkout ${{ github.event.release.tag_name }}
         uses: actions/checkout@v4.1.1
         with:
-          ref: ${{ github.event.release.tag_name || github.event.inputs.tag_name }}
+          ref: ${{ github.event.release.tag_name }}
           lfs: true
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
       - name: Log in to GitHub Container Registry
         uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Generate Docker tags
-        id: meta
-        uses: docker/metadata-action@v5
-        with:
-          images: ghcr.io/${{ github.repository }}
-          tags: |
-            type=semver,pattern={{version}},value=${{ github.event.release.tag_name || github.event.inputs.tag_name }}
-            type=semver,pattern={{major}}.{{minor}},value=${{ github.event.release.tag_name || github.event.inputs.tag_name }}
-            latest
-
-      - name: Configure QEMU
-        uses: docker/setup-qemu-action@v3
-
-      - name: Configure Buildx
-        uses: docker/setup-buildx-action@v3
+      - name: Format version
+        id: format_version
+        run: |
+          VERSION=${{ github.event.release.tag_name }}
+          FORMATTED_VERSION=$(echo "$VERSION" | sed 's/^v//')
+          echo "::set-output name=formatted_version::$FORMATTED_VERSION"
 
-      - name: Build and push
+      - name: Build and Push Docker image
```

Comment:
I would recommend to keep it one of those both ways

- Build and Push Docker Image
- Build and puh Docker image